### PR TITLE
Migration to yarn & test monorepo

### PR DESCRIPTION
**Pull request by kamil-stasiak**

### CODE REVIEW

File: assets/src/features/devices/LocalPeerMediaContext.tsx [hunk 1]
```diff
@@ -170,6 +170,33 @@ export const LocalPeerMediaProvider = ({ children }: Props) => {
...
+      }
+    };
+
+    const localTrackMuted: ClientEvents<PeerMetadata, TrackMetadata>["localTrackMuted"] = async (event, clientApi) => {
+      const prevMetadata = client.local?.tracks?.[event.trackId].metadata;
+
+      if (clientApi.status === "joined" && prevMetadata) {
+        client.updateTrackMetadata(event.trackId, { ...prevMetadata, active: false });
+      }
+    };
```
Comment (Rados13) [inline assets/src/features/devices/LocalPeerMediaContext.tsx:188]: Why do we need that change?

**Review by Rados13 — APPROVED** [review]

**Comment by karkakol** [comment]
image

File: assets/package.json [hunk 2]
```diff
@@ -11,7 +11,7 @@
     "preview": "vite preview"
   },
   "dependencies": {
-    "@fishjam-dev/react-client": "github:fishjam-dev/react-client-sdk#main",
+    "@fishjam-dev/react-client": "https://github.com/fishjam-cloud/web-client-sdk.git#workspace=@fishjam-dev/react-client&head=fix-error-on-muted-track-renegotiation",
```
Comment (karkakol) [inline assets/package.json:14]: This should be `nice` link. Change it before merging

File: assets/package.json [hunk 3]
```diff
@@ -33,6 +33,12 @@
     "uuid": "^9.0.1",
     "zod": "^3.22.4"
   },
+  "resolutions": {
+    "@fishjam-dev/ts-client": "https://github.com/fishjam-cloud/web-client-sdk.git#workspace=@fishjam-dev/ts-client&head=fix-error-on-muted-track-renegotiation"
+  },
+  "overrides": {
+    "@fishjam-dev/ts-client": "https://github.com/fishjam-cloud/web-client-sdk.git#workspace=@fishjam-dev/ts-client&head=fix-error-on-muted-track-renegotiation"
```
Comment (karkakol) [inline assets/package.json:40]: same as above

File: assets/src/features/devices/LocalPeerMediaContext.tsx (hunk 1 above)
Comment (kamil-stasiak) [reply assets/src/features/devices/LocalPeerMediaContext.tsx:188]: To display track muted icon

**Review by karkakol — APPROVED** [review]

**Pull request merged**